ocfweb/docs/docs/services/web.md

```diff
@@ -11,6 +11,52 @@ where `user` is the account name.
 Group accounts may also request a [[virtual host|doc services/vhost]] for
 another domain (e.g., group.berkeley.edu).
 
+#### Off-site hosting
```

Comment:
Can we put most of this information somewhere else? A small section here that links to another page might work well. Right now this takes a page which contains useful technical information about updating your website and inserts a giant blob of political text which isn't relevant to most people.